js/ui/overlay/ui.overlay.js

```diff
@@ -1272,21 +1272,44 @@ var Overlay = Widget.inherit({
 
     _renderShading: function() {
         this._fixWrapperPosition();
+        if(this.option("shading")) {
+            this._renderShadingDimensions();
```

Comment:
If the size of the popup is set by relative values, it will be displayed correctly only when shading is enabled:
![image](https://user-images.githubusercontent.com/1554153/66482926-b4329880-eaac-11e9-82e8-69678a9fc560.png)
![image](https://user-images.githubusercontent.com/1554153/66482970-c9a7c280-eaac-11e9-86ed-493ce43d4393.png)